translate/cloud-client/src/main/java/com/example/translate/HybridGlossaries.java

```diff
@@ -0,0 +1,301 @@
+ * limitations under the License.
+ */
+
+package com.example.translate;
+
+// [START translate_hybrid_imports]
+import com.google.cloud.texttospeech.v1.AudioConfig;
+import com.google.cloud.texttospeech.v1.AudioEncoding;
+import com.google.cloud.texttospeech.v1.SsmlVoiceGender;
+import com.google.cloud.texttospeech.v1.SynthesisInput;
+import com.google.cloud.texttospeech.v1.SynthesizeSpeechResponse;
+import com.google.cloud.texttospeech.v1.TextToSpeechClient;
+import com.google.cloud.texttospeech.v1.VoiceSelectionParams;
+
+import com.google.cloud.translate.v3beta1.CreateGlossaryRequest;
+import com.google.cloud.translate.v3beta1.GcsSource;
+import com.google.cloud.translate.v3beta1.Glossary;
+import com.google.cloud.translate.v3beta1.Glossary.LanguageCodesSet;
+import com.google.cloud.translate.v3beta1.GlossaryInputConfig;
+import com.google.cloud.translate.v3beta1.GlossaryName;
+import com.google.cloud.translate.v3beta1.LocationName;
+import com.google.cloud.translate.v3beta1.TranslateTextGlossaryConfig;
+import com.google.cloud.translate.v3beta1.TranslateTextRequest;
+import com.google.cloud.translate.v3beta1.TranslateTextResponse;
+import com.google.cloud.translate.v3beta1.TranslationServiceClient;
+
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.Feature.Type;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+
+import com.google.common.html.HtmlEscapers;
+import com.google.protobuf.ByteString;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+// [END translate_hybrid_imports]
+
+
+public class HybridGlossaries {
+
+  // [START translate_hybrid_project_id]
+  private static final String projectId = System.getenv("PROJECT_ID");
+  // [END translate_hybrid_project_id]
+
+  // [START translate_hybrid_vision]
+  /**
+   * @param filePath input image file
+   **/
+  static String picToText(String filePath) {
+    List<AnnotateImageRequest> requests = new ArrayList<>();
+
+    try (ImageAnnotatorClient client = ImageAnnotatorClient.create()) {
+      ByteString imageBytes = ByteString.readFrom(new FileInputStream(filePath));
+
+      Image image = Image.newBuilder().setContent(imageBytes).build();
+
+      Feature feature = Feature.newBuilder().setType(Type.DOCUMENT_TEXT_DETECTION).build();
+
+      AnnotateImageRequest request =
+              AnnotateImageRequest.newBuilder()
+                      .addFeatures(feature)
+                      .setImage(image)
+                      .build();
+
+      requests.add(request);
+
+      BatchAnnotateImagesResponse response = client.batchAnnotateImages(requests);
```

Comment:
Do we need to use Batch? Is there an API for a single image call?

If not, we can at least inline requests:
```suggestion
      BatchAnnotateImagesResponse response = client.batchAnnotateImages(Arrays.asList(request));
```

Reply:
Yea, there is only batch or asynchronous calls. But updated to use `Arrays.asList`